### DISCUSSION TITLE
请问大神GPT怎么打开和设置呢？

### DISCUSSION THREAD

**Discussion by s23ir45** [MuiseDestiny/zotero-style#211] [General]
大神，请问Zotero Style 中GPT怎么打开和设置呢？

**Comment by MuiseDestiny** [comment]
打开Zotero，
![image](https://user-images.githubusercontent.com/51939531/223344560-597851a1-c88a-4f64-bdbc-cde76af0fdf2.png)

填入后，`Shift + Tab + P`
![image](https://user-images.githubusercontent.com/51939531/223344742-318b68a0-59fe-44d0-8a6d-80a111b7c095.png)

输入`>`+一个空格
![image](https://user-images.githubusercontent.com/51939531/223344875-57e6435a-c8ef-428d-80c3-e58512c98fe7.png)

输入你的问题，回车
![image](https://user-images.githubusercontent.com/51939531/223345033-7c5735ae-eba7-4a2a-ae3d-6576d6d979c1.png)

你就得到了一个它瞎编的答案
![image](https://user-images.githubusercontent.com/51939531/223345090-05af8c29-9a96-4546-80a8-55b5f837a62e.png)

**Comment by Wenwen-ai-cell** [comment]
![image](https://user-images.githubusercontent.com/78087414/225792129-e169a586-d3dd-4c71-b922-9dd5e7d9f867.png)
请问这个地方是填什么呀，已有账号密码，但是就是不知道这个地儿咋填，能麻烦知道的人告诉一下孩子嘛
